src/templates/components/home.hbs

```diff
@@ -0,0 +1,5 @@
+<div id="home" class="home">
+    {{> components/sidebar}}
```

Comment:
Add header component and move logos to here

Reply:
Done